devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -14,7 +14,8 @@ import groovy.json.JsonOutput
  *
  */
 metadata {
-    definition (name: "Arrival Sensor HA", namespace: "smartthings", author: "SmartThings") {
+    definition (name: "Arrival Sensor HA", namespace: "smartthings", author: "SmartThings",
+            runLocally: true, minHubCoreVersion: '000.025.0000', executeCommandsLocally: true) {
```

Comment:
@varzac , what version first supported this device locally?

Reply:
https://smartthings.atlassian.net/browse/CHAD-882 that and the changelog say 18.XX